apps/actors/lib/actors/actor/entity/entity.ex

```diff
@@ -533,6 +600,39 @@ defmodule Actors.Actor.Entity do
        ),
        do: timeout + timeout_factor
 
+  defp handle_schedulers(schedulers) when is_list(schedulers) do
+    schedulers
+    |> Flow.from_enumerable()
+    |> Flow.map(fn %ScheduledTimerCommand{expression: expression, command: command} = cron_command ->
+      Logger.debug("Registering cron command #{inspect(cron_command)}")
+      delay = get_time_from_cron(expression)
+      Process.send_after(self(), {:invoke_timer_command, command}, delay)
+    end)
+    |> Flow.run()
+  end
+
+  defp handle_schedulers(nil), do: :ok
+
+  defp handle_schedulers([]), do: :ok
```

Comment:
this will never reach because [] is a list

Reply:
True